common/lib/xmodule/xmodule/tabs.py

```diff
@@ -789,43 +205,47 @@ def initialize_default(course):
         """
 
         course.tabs.extend([
-            CoursewareTab(),
-            CourseInfoTab(),
+            CourseTab.from_json({'type': 'courseware', 'name': _('Courseware')}),
+            CourseTab.from_json({'type': 'course_info', 'name': _('Course Info')})
```

Comment:
Optional: This feels odd to me. I think I'd rather see these written out as something like `CourseTab.load('courseware', name=_('Courseware'))` (which I don't think you have), or perhaps just by assembling the default list of json blobs, and then calling `from_json` on all of them at the end of the method.